okta/UsersClient.py

```diff
@@ -49,18 +49,24 @@ def update_user(self, user):
         :type user: User
         :rtype: User
         """
+
```

Comment:
nit: we don't add newlines after the doc tags in other clients, so we should be consistent.